File: Linked List/Linked List in Python/linkedlist.py
# Linked List in Python

class node:
    def __init__(self,data):
        self.data=data
        self.next=None

class LinkedList:
    def __init__(self):
        self.start=None

    def view(self):
        if(self.start==None):
            print("Empty list\n")
        else:
            temp=self.start
            while(temp!=None):
                print(temp.data, end=" ")
                temp=temp.next
            print()
    
    def DeleteFromStart(self):
        if self.start==None:
            print("List is empty. Nothing to delete!\n")
        else:
            print("Deleted value is:", self.start.data)
            self.start=self.start.next

    def InsertInEnd(self,value):
        newNode=node(value)
        if(self.start==None):
            self.start=newNode
        else:
            temp=self.start
            while(temp.next != None):
                temp=temp.next
            temp.next=newNode

# Driver Menu
ch=0
myList=LinkedList()
while(ch!=4):
    print("1. Insert at End\n2. Delete from Start\n3. Display\n4. Exit")
    ch=int(input())
    if(ch==1):
        print("Enter Value to insert:")
        n=int(input())
        myList.InsertInEnd(n)
    elif(ch==2):
        myList.DeleteFromStart()
    elif(ch==3):
        myList.view()
    elif(ch==4):
        exit()        

File: Linked List/Linked List in Java/LinkedList.java
import java.lang.*;
import java.util.*;

class LinkedList
{
    Node head;
    static class Node
    {
        Node next;
        int data;
        Node(int value)
        {
            data = value;
            next = null;
        }
    }

    static LinkedList insert(LinkedList list, int value)
    {
        Node new_node = new Node(value);
        
        if(list.head == null)
            list.head = new_node;
        else
        {
            Node temp = list.head;
            while(temp.next != null)
                temp = temp.next;
            temp.next = new_node;
        }
        return list;
    }

    static void print(LinkedList list)
    {
        if(list.head == null)
            System.out.println("Linkedlist is Empty!");
        else
        {
            System.out.println("Linked List is:");
            Node temp = list.head;
            while(temp != null)
            {
                System.out.print(temp.data + "->");
                temp = temp.next;
            }
            System.out.print("End");
        }
    }

    public static void main(String args[])
    {
        LinkedList list = new LinkedList();
		list = insert(list, 1); 
		list = insert(list, 2); 
		list = insert(list, 3); 
		list = insert(list, 4); 
		list = insert(list, 5); 
		list = insert(list, 6); 
		list = insert(list, 7); 
		list = insert(list, 8);
        print(list);
    }
}